app/scripts/services/Room.js

```diff
@@ -3,8 +3,14 @@
 		var ref = firebase.database().ref().child("rooms");
 		var rooms = $firebaseArray(ref);
 
+		var addRoom = function(roomName) {
+			rooms.$add(roomName);
+			roomnName = null;
```

Comment:
This is misspelled so it won't do anything but anyway, it doesn't belong here. I assume you're using this to try and empty out the `input` tag but that kind of action belongs in the controller, not the service.